build.sbt

```diff
@@ -92,3 +93,24 @@ osgiSettings
 OsgiKeys.exportPackage  := Seq("akka.persistence.cassandra.*")
 OsgiKeys.importPackage  := Seq(akkaImport(), optionalImport("org.apache.cassandra.*"), "*")
 OsgiKeys.privatePackage := Seq()
+
+lazy val cassandraLauncher = (project in file("cassandra-launcher"))
+  .settings(common: _*)
+  .settings(
+    name := "akka-persistence-cassandra-launcher",
+    managedResourceDirectories in Compile += (target in cassandraBundle).value / "bundle",
+    managedResources in Compile += (assembly in cassandraBundle).value
+  )
+
+// This project doesn't get published directly, rather the assembled artifact is included as part of cassandraLaunchers
+// resources
+lazy val cassandraBundle = (project in file("cassandra-bundle"))
+  .settings(
+    name := "akka-persistence-cassandra-bundle",
+    crossPaths := false,
+    autoScalaLibrary := false,
+    libraryDependencies += "org.apache.cassandra" % "cassandra-all" % "3.10" exclude("commons-logging", "commons-logging"),
```

Comment:
No we don't, because we just produce one fat jar with all the dependencies inside it, and that jar is included as a resource inside the launcher jar, so its classpath is completely separated from and unrelated to whatever is using it.